### PR TITLE
"Update chart-features-trendlines.md""

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd seo trendlines
using fc

### CODE REVIEW

File: doc/en/components/charts/chart-features-trendlines.md [hunk 1]
```diff
@@ -7,7 +7,87 @@ mentionedTypes: []
...
+The $PlatformShort$ financial stock chart has support for trendlines, which help to identify a trend in data bound to a series.
 
-data-chart-series-trendlines.md
--->
\ No newline at end of file
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/data-chart-series-trendlines" 
+           alt="$PlatformShort$ Trendlines Example" 
+           github-src="charts/data-chart/series-trendlines">
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-trendlines.md:16]: why CV is pointing to DataChart samples and not FinancialChart sample?

File: doc/en/components/charts/chart-features-trendlines.md [hunk 2]
```diff
@@ -7,7 +7,87 @@ mentionedTypes: []
...
+- `SimpleAverage`
+- `WeightedAverage`
+
+The following code snippet demonstrates how to add a trendline to a series in the $PlatformShort$ data chart component:
+
+```razor
+<FinancialChart 
+    Width="100%"
+    Height="100%"
+    ChartType=FinancialChartType.Bar
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-trendlines.md:46]: missing double quotes for value of ChartType prop

File: doc/en/components/charts/chart-features-trendlines.md [hunk 3]
```diff
@@ -7,7 +7,87 @@ mentionedTypes: []
...
+    TrendLineBrushes="rgba(0, 101, 209, 1)"/>
+```
+
+```html
+<igx-financial-chart 
+    width="100%"
+    height="100%"
+    chartType="Bar"
+    dataSource="[DataSource]"
+    trendLineType="@TrendLineType"
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-trendlines.md:61]: set trend line type to "QuinticFit"

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/chart-features-trendlines.md (hunk 1 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-trendlines.md:16]: pending changes

File: doc/en/components/charts/chart-features-trendlines.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-trendlines.md:46]: pending

File: doc/en/components/charts/chart-features-trendlines.md (hunk 3 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-trendlines.md:61]: pending

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**